test/cypress/support/commands.ts

```diff
@@ -357,6 +357,8 @@ const commands = {
             });
     },
 
+    clearSearchableDropdown: (fieldName: string) => cy.contains('.field', fieldName).find('.dropdown.icon').click(),
```

Comment:
IIUC, with the current implementation, `clearSearchableDropdown` will also work on a dropdown which does not have clear icon (non-clearable dropdown or no value set in clearable dropdown).

I'm wondering shouldn't we be more specific and mention `clear` class as well?
```suggestion
    clearSearchableDropdown: (fieldName: string) => cy.contains('.field', fieldName).find('.dropdown.clear.icon').click(),
```

Reply:
Good point :+1: Otherwise, we could this `.dropdown.icon`:
![image](https://user-images.githubusercontent.com/889383/126953528-b576b301-188f-4e65-a8f4-043356a7895e.png)

Thanks for catching that, I'll change it according to your suggestion

Reply:
I could not commit your GitHub suggestion, unfortunately, as the formatting would be invalid :worried: Anyway, done :slightly_smiling_face: